SU-SWS/decanter
# D8CORE-6976: removed italics from the quote style

# READY FOR REVIEW

# Summary
- Removed the italics style from the quote-text mixin. This was done for a11y reasons.

# Needed By (Date)
- as time allows

# Urgency
- normal

# Steps to Test

1. Checkout this branch.
2. Use the quote-text mixin.
3. Verify it is not coming with italics.

# Affected Projects or Products
- Sites using Decanter and the Stanford Basic theme

# Associated Issues and/or People
[- D8CORE-6976](https://stanfordits.atlassian.net/browse/D8CORE-6976)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
